extensions/amp-form/0.1/amp-form-textarea.js

```diff
@@ -113,11 +113,12 @@ export class AmpFormTextarea {
     this.unlisteners_.push(listen(root, AmpEvents.DOM_UPDATE, () => {
       cachedTextareaElements = root.querySelectorAll('textarea');
     }));
-    const throttledResize = throttle(this.win_, e => {
-      if (e.relayoutAll) {
-        resizeTextareaElements(cachedTextareaElements);
-      }
-    }, MIN_EVENT_INTERVAL_MS);
+    const throttledResize = throttle(
+        /** @type {!Window} */ (this.win_), e => {
```

Comment:
type should probably be in declaration of the field in constructor instead of usage time